wildstore-cli/src/main/java/edu/sjsu/wildstore/Main.java

```diff
@@ -123,32 +123,31 @@ public void search(@CommandLine.Parameters(paramLabel = "query") String query,
         }
 
         @CommandLine.Command
-        public void clean(@CommandLine.Parameters(paramLabel = "limit") int limit,
-                          @CommandLine.Parameters(paramLabel = "hostname") String hostname,
-                          @CommandLine.Option(names = "--token") String token,
-                          @CommandLine.Option(names = "--dryrun", negatable = true, defaultValue = "true")
-                          boolean dryrun) throws InterruptedException, ExecutionException {
+        public void clean(@CommandLine.Mixin CliOptions co,
+                          @CommandLine.Parameters(paramLabel = "limit", defaultValue = "10000") int limit,
+                          @CommandLine.Option(names = "--dryrun", negatable = true) Boolean dryrun)
```

Comment:
i think we want to keep the default value to true (this can be a dangerous operation, so we need to make sure..)

Reply:
i'm hoping that if you supply a default value, you can use boolean dryrun (lowercase b) :)